sw/device/lib/dif/autogen/meson.build

```diff
@@ -58,6 +58,20 @@ sw_lib_dif_autogen_csrng = declare_dependency(
   )
 )
 
+# Autogen EDN DIF library
+sw_lib_dif_autogen_edn = declare_dependency(
+  link_with: static_library(
+    'sw_lib_dif_autogen_edn',
+    sources: [
+      hw_ip_edn_reg_h,
+      'dif_edn_autogen.c',
+    ],
+    dependencies: [
+      sw_lib_mmio,
+    ],
+  )
+)
+
```

Comment:
nit: does this change belong to this PR?

Reply:
This gets rendered weird in the diff, really this was not added, it was just moved to be in alpha-order with rest of existing targets.